packages/react-examples/src/react-experiments/UnifiedPeoplePicker/UnifiedPeoplePicker.WithEdit.Example.tsx

```diff
@@ -149,7 +149,7 @@ export const UnifiedPeoplePickerWithEditExample = (): JSX.Element => {
   const _isValid = React.useCallback((item: IPersonaProps): boolean => Boolean(item.secondaryText), []);
 
   const SelectedItemInternal = (props: ISelectedItemProps<IPersonaProps>) => (
-    <SelectedPersona isValid={_isValid} {...props} />
+    <SelectedPersona isValid={_isValid} {...props} size={PersonaSize.size32} />
```

Comment:
nit: would probably be good for the example to be different than the default

Reply:
I sorta meant to revert this after the first iteration. Just included it to show where I set the value.

Reply:
I think it would actually be good for one of the examples to have a different size than the default.

Reply:
Done, made the example slightly larger "size48" which is stated in the comment as the default size..... I think that comment is out of date (or we should change the default from 32 to 48 to match the comment :)